liquibase-standard/src/main/java/liquibase/datatype/core/BigIntType.java

```diff
@@ -54,16 +54,15 @@ public DatabaseDataType toDatabaseDataType(Database database) {
             return new DatabaseDataType("BIGINT");
         }
         if (database instanceof PostgresDatabase) {
-            if (isAutoIncrement()) {
-                if (((PostgresDatabase) database).useSerialDatatypes()) {
-                    return new DatabaseDataType("BIGSERIAL");
+            if (isAutoIncrement() && ((PostgresDatabase) database).useSerialDatatypes()) {
+                return new DatabaseDataType("BIGSERIAL");
+            } else {
```

Comment:
Before this else would execute only if it was autoIncrement, but now it always executes for postgresql. Is it expected?

Reply:
Yes this is correct. For Posgres is it never allowed to use the default behavior (using BIGINT with size)

Reply:
Thanks!